package.json

```diff
@@ -66,6 +66,7 @@
     "through2": "~2.0.0"
   },
   "files": [
-    "lib"
+    "lib",
+    "bin"
```

Comment:
BTW [second time](https://github.com/gruntjs/grunt-contrib-internal/commit/3a9866afad85de6ac5472d0e5b2b2a0e4786e5d0) we got bitten by this today :/